### ISSUE TITLE
Version 2 for long-reads

### ISSUE THREAD

**Issue by fmaguire** [jaleezyy/covid-19-signal#45]
Bigger milestone to start as separate repo but version of pipeline to support minion and pacbio

**Comment by jaleezyy** [comment]
I can always make that repo whenever. :)

**Comment by fmaguire** [comment]
As soon as we have data, make that repo, and we can close this stub issue!

**Comment by jts** [comment]
I strongly recommend that if you add support for MinION data you only wrap the `artic` commands from the [SOP](https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html) in the same style as the [nextflow wrapper](https://github.com/connor-lab/ncov2019-artic-nf/blob/master/workflows/articNcovNanopore.nf). There's been a lot of work to get primer trimming right for nanopore data and handling the two amplicon pools correctly and it would be challenging to start from scratch

**Comment by fmaguire** [comment]
Yeah, that is pretty much what were thinking for the core components @jts  definitely don't want to reinvent the wheel.

**Comment by passdan** [comment]
Was wondering if you did any more on supporting nanopore through the signal pipeline? We've been using the pipeline for illumina (works great!) and wanted to touch some nanopore. 

As far as I can see there's not an option but wanted to check if there was before I jump ship to the artic nextflow for doing the nanopore data. Thanks!

**Comment by jaleezyy** [comment]
At present, we don't have a version for Nanopore. The ARTIC Nextflow pipeline really goes a long way for long-read sequencing.
